### ISSUE TITLE
Feature request: Baseline / best practices recipe

### ISSUE THREAD

**Issue by turian** [iver56/audiomentations#212]
It would be nice to have a "recipe" feature, so you could use a predefined set of audiomentations.

I would propose to start with a simple baseline that implements safe "best practices" that aren't radical. So that there is a quickstart one-liner for people who are paralyzed by the agony of many choices and hyperparameters.

**Comment by turian** [comment]
https://iopscience.iop.org/article/10.1088/1742-6596/1453/1/012085/pdf

<img width="589" alt="image" src="https://user-images.githubusercontent.com/65918/184522850-df34d074-7a26-4428-acbc-c233195e7bde.png">

**Comment by iver56** [comment]
I like the idea. I guess one might need different recipes for different kinds of problems then

**Comment by turian** [comment]
@iver56 yeah but the idea would be that there would be a "general" recipe for unknown problems.
